targeted/cve-2021-22214.js

```diff
@@ -51,21 +49,26 @@ function invokeWith(msg) {
     // Checks to make sure that the response is by a Gitlab Instance
     // on the server
     if (status === 200 && ctype === "application/json") {
-        // A unqiue term in the demo YAML file which will be present in
+        // A unique term in the demo YAML file which will be present in
         // the response body when the lint check succeeds
         if (rebody.contains("CI_PIPELINE_SOURCE")) {
+            // risk: 0: info, 1: low, 2: medium, 3: high
+            // confidence: 0: falsePositive, 1: low, 2: medium, 3: high, 4: confirmed
+            alertParam = url + "/api/v4/ci/lint";
+            alertAttack = String(exploitResponse.getRequestBody().toString())
+            alertEvidence = "Presence of a unique term, [CI_PIPELINE_SOURCE] was detected in the response, which is part of the payload."
```

Comment:
- Param is intended to literally be the impacted param (or header field in the case of some passive scripts) If the issues manifests via a simple GET request then there's no param implicated.
- Attack should be the actual payload sent to trigger the issue.
- Evidence should be a string in the request/response which validates or indicates the issue is present, and will be highlighted when someone is looking at alerts in the GUI.

Reply:
You could include this "alertEvidence" sentence in "Other Info" (or description) if you feel it will help people.

Reply:
Have modified accordingly.